Cookbook/Technical-Documents/AddingFrameworks.md

```diff
@@ -10,11 +10,12 @@ If you need to create a new framework target in our project to add a new module,
 
 _The following steps will guide you to configure a new module named `BabylonNewFramework`. Replace every occurrence of `BabylonNewFramework` in the following instructions with the actual name of your new internal framework._
 
-### Add the framework and its dependencies in Podfile
+Start by creating the new framework target (and its associated Test target(s)) in Xcode, then follow the steps below:
 
-1. Create the framework target (and its associated Test target(s))
-2. Add the framework target (and its test target) to the `Podfile`, listing all its third-party dependencies (like `Bento`, `ReactiveCocoa`, `ReactiveFeedback`, etc) in there
-3. Run `bundle exec pod install` to install the framework's pods
+### Add Third-Party dependencies in the Podfile
+
+1. Add the framework target (and its test target(s)) to the `Podfile`, listing all its third-party dependencies (like `Bento`, `ReactiveCocoa`, `ReactiveFeedback`, etc) in there
+2. Run `bundle exec pod install` to install the framework's pods
```

Comment:
ℹ️ Splitting that in two (extracting out the first point about "Creating the framework target") helped when telling to reuse those same steps on [L81](https://github.com/Babylonpartners/ios-playbook/pull/141/files#diff-484da85a726376a689dcda46e0a270dfR81) down below